src/compute.ts

```diff
@@ -168,3 +176,52 @@ export const encryptedVolumes: ResourceValidationPolicy = {
     }),
 };
 registerPolicy("encryptedVolumes", encryptedVolumes);
+
+/** @internal */
+export const securityGroupNoRuleManagementConflicts: StackValidationPolicy = {
+  name: "security-group-no-rule-management-conflicts",
+  description:
+    "Checks that ec2.SecurityGroup resources do not conflict with vpc.SecurityGroupEgressRule, vpc.SecurityGroupIngressRule, or ec2.SecurityGroupRule.\n"+
```

Comment:
The description gets printed by the CLI output, not sure how good multi-line looks there but probably readable.